AdaptiveCards/templating/index.md

```diff
@@ -12,7 +12,16 @@ We're excited to share a preview of new tools that will help you **create**, **r
 
 > [!IMPORTANT] 
 > 
-> These features are **in preview and subject to change**. Your feedback is not only welcome, but  critical to ensure we deliver the features **you** need.
+> **Breaking changes** in the **May 2020 Release Candidate**
+>
+> We've been hard at work getting templating released, and we're finally in the home stretch! We had to make some minor breaking changes as we close on the release.
+
+## Breaking changes as of May 2020
+
+1. The binding syntax changed from `{...}` to `${...}`. 
+    * For Example: `"text": "Hello {name}"` becomes `"text": "Hello ${name}"`
+2. The JavaScript API no longer contains an `EvaluationContext` object. Simply pass your data to the `expand` function. Please see the [SDK page](sdk.md) for full details.
+3. The .NET API was redesigned to more closely match the JavaScript API. Please see the [SDK page](sdk.md) for full details.
 
 ## How can templating help you?
```

Comment:
This is new learning for me; Per [Microsoft Writing recommendation here](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28361) skip punctuation at the end of the title/subtitle etc (!)